packages/ckeditor5-utils/src/madewith.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module utils/madewith
+ */
+
+/**
+ * Possible states of the key after verification
+ */
+export type VerifiedKeyStatus = 'VALID' | 'INVALID' | 'EXPIRED';
+
+/**
+ * Checks whether the given string contains information that allows you to verify the license status.
+ *
+ * @param stringToCheck The string to check.
+ * @returns String that represents the state of given `stringToCheck` parameter. It can be `'VALID'`, `'INVALID'` or `'EXPIRED'`.
+ */
+export default function verify( stringToCheck: string ): VerifiedKeyStatus {
+	// This is just a very simplified preliminary front-end check of the date validation with the current release date - it
```

Comment:
What's the purpose of this comment?

Reply:
I just wanted to add initial version of "friendly comment" ( https://github.com/cksource/ckeditor5-internal/issues/3164#issuecomment-1516053695 ).